instrumentation/dubbo/dubbo-apache-2.7/library/src/test/groovy/io/opentelemetry/instrumentation/dubbo/apache/v2_7/DubboTest.groovy

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.dubbo.apache.v2_7
+
+import io.opentelemetry.instrumentation.dubbo.apache.v2_7.api.HelloService
+import io.opentelemetry.instrumentation.dubbo.apache.v2_7.impl.HelloServiceImpl
+import io.opentelemetry.instrumentation.test.InstrumentationTestTrait
+import org.apache.dubbo.config.ReferenceConfig
+import org.apache.dubbo.config.RegistryConfig
+import org.apache.dubbo.config.ServiceConfig
+
+class DubboTest extends AbstractDubboTest implements InstrumentationTestTrait {
+  @Override
+  ServiceConfig configureServer() {
+    def registerConfig = new RegistryConfig()
+    registerConfig.setAddress("N/A")
+    ServiceConfig<HelloServiceImpl> service = new ServiceConfig<>()
+    service.setInterface(HelloService)
+    service.setRef(new HelloServiceImpl())
+    service.setRegistry(registerConfig)
+    service.setFilter("OtelServerFilter")
```

Comment:
Is setFilter required even though we have the resource files?

Reply:
I found that if I add the Activate annotation at the beginning of a class it will be loaded automatically, and it is worked well in library.But maybe Dubbo's spi is more complicated than I thought, if I inject the resource file directly, it will be something wrong in javaagent's unit test.It will be the exception like failOnContextLeak I've spent a lot of time checking, but I don't have a clue.It seems that the process of Dubbo loading the server filter is different from that of the client filter.I have changed the code in javaagent in order to make it work by Activate annotation, and it can avoid the exception failOnContextLeak now in javaagent's unit test.